libs/challenge-registry/user-profile/src/lib/user-profile-starred/user-profile-starred.component.html

```diff
@@ -31,7 +31,7 @@ <h3>Starred</h3>
           [pageLinkSize]="5"
           [rows]="limit"
           [totalRecords]="challengeResultsCount"
-          [rowsPerPageOptions]="[10, 20, 30]"
+          [rowsPerPageOptions]="[12, 24, 36]"
           [showCurrentPageReport]="true"
           currentPageReportTemplate="Showing {first} to {last} of {totalRecords} results"
           (onPageChange)="onPageChange($event)"
```

Comment:
Sorry not able to tag in the right lines. Can you also add `card-group` to user-starred's organization cards? Thank you!

Reply:
Good catch! Forgot about that page.